georel/srv/service.js

```diff
@@ -9,7 +9,7 @@ const { BusinessPartner, BusinessPartnerRole, BusinessPartnerAddress, AddressPho
 const RELEVANT_COUNTRY = process.env.RELEVANT_COUNTRY || 'US'
 const RELEVANT_ROLE = 'FLCU01'
 
-const NAMESPACE = 'sap/S4HANAOD/1908'
+const NAMESPACE = 'sap/S4HANAOD/0422'
```

Comment:
Why do we need a new number here?

Reply:
this is was my mistake, you can have the older number